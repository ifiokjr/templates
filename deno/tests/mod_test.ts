import { describe, it } from "./deps.ts";
import { snapshot } from "./helpers.ts";

describe("main", () => {
  it("should create snapshots", async (t) => {
    await snapshot(t, { hello: "world" });
  });
});
