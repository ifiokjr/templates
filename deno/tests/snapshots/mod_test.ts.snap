export const snapshot = {};

snapshot[`main > should create snapshots 1`] = `{ hello: "world" }`;
